### ISSUE TITLE
Contributors needed for an "Introduction to Hyperledger fabric" overview doc

### ISSUE THREAD

**Issue by joshhus** [hyperledger-archives/fabric#1491]
## Description

Current fabric/docs doc lacks "big picture" overview of the solution. Proposing some adds for "Introduction to Hyperledger fabric" documentation here. Please feel free to add / change / edit / add new sections as you see fit:

Proposed text for a "Welcome" page (note words in bold will be linked to the glossary definitions):
## Welcome to Hyperledger fabric

The Linux Foundation Hyperledger Project(1) is the result of collaboration between project members(2) who share the common goal of designing a business-ready, open-source standard for blockchain networks. Extensive research, planning and implementation by this community has led to the Linux Foundation adopting Hyperledger fabric as its open source standard for blockchain network architecture.

The Hyperledger fabric blockchain standard is a base architecture, with built-in functionality, security and platform independence for blockchain networks. The open standard facilitates adoption by any public or private organization or enterprise, and in turn by their constituent members. 

References: 
(1) Hyperledger Project: https://www.hyperledger.org/ 
(2) Hyperledger Project members: https://www.hyperledger.org/about/members
### Audience

The audience for this Hyperledger fabric documentation includes members of any public or private group, organization or enterprise who are interested in joining the Linux Foundation Hyperledger Project and adopting its open standard for their business networks. The organizational roles that this audience fulfills are diverse, and include business executives and decision-makers, system  architects, network designers, application developers, security administrators, auditors and  regulators. 

Any potential end user of a blockchain business network would also benefit from this documentation, including professionals at securities and commodities exchanges, manufacturing plants, distribution centers, leasing agencies, business service providers and government agencies.
### Prerequisites

The only prerequisites to employ Hyperledger fabric for blockchain networks are an interest in, and the resources to support, a distributed business network that processes transactions with significant 
decreases in cost, increases in speed and enhancements in security, compared to traditional networks. Though not required, most adopters of Hyperledger fabric will already be using one or more active business networks; integration of these existing networks with Hyperledger fabric is a fully-supported business service.

All sizes of business networks are supported by Hyperledger fabric, from a single node to networks 
containing hundreds of nodes. Typical hardware and software networking components, and resources to develop applications and write chaincode, are basic requirements.
## Describe How to Reproduce

The current fabric/docs are lacking "big picture" descriptions of the Hyperledger Project blockchain solution.

**Comment by joshhus** [comment]
Taking this one ...

**Comment by ghost** [comment]
Josh, 

If there's going to be an introductory section on "what is blockchain" I'd like to take it on. It seems like every explainer out there suffices itself to explain _what_ blockchain is, not _why_ it is. 

It's a complicated subject to boil down without getting bogged down in historical quicksand but when I first started to learn about blockchain I was immensely frustrated by the lack of context in everything I read. So-called blockchain "explainers" always reduced themselves to a kind of tautology; you could only understand their explanation of what blockchain was once you already understood what blockchain was. 

This might be too broad for the document you're envisioning. Let me know. I'm not sure the degree to which the group of people you're targeting with the document will already understand what a "ledger" is, much less what a peer-to-peer automatically distributed ledger is or does.

**Comment by joshhus** [comment]
Yes Joe, I like the sound of it; especially the _why_ blockchain. ... Broad / big picture documents are what folks say is missing, so go ahead and draft something please ... can be plain text, or .md. ... Use a new issue for this though. I'm adding the "Welcome" above and will close it. I also have other sections drafted (not complete / ready necessarily), which I will email you.

**Comment by joshhus** [comment]
Are you on our slack channels?

**Comment by ghost** [comment]
Josh,

No I'm not. How do I subscribe?

**Comment by joshhus** [comment]
Try the "Slack" link here: https://www.hyperledger.org/community

**Comment by ghost** [comment]
Ok. Joined and subscribed to the "fabric" and "docs" channels.

**Comment by joshhus** [comment]
I am joshhus - try to send me a msg.
